Samples/CollectAndRevealSample/CollectAndRevealSample/ViewController.swift

```diff
@@ -174,10 +175,15 @@ class ViewController: UIViewController {
     }
     internal func updateSuccess(_ response: SuccessResponse) {
         print(response)
+        retryCount = 0
         updateRevealInputs(tokens: response.records[0].fields)
         print("Successfully got response:", response)
     }
-    internal func updateFailure() {
+    internal func updateFailure(error: Any) {
+        if((error as AnyObject).contains("Invalid Bearer token") && retryCount <= 2){
```

Comment:
Just add a todo, it will be replaced with error code in the future

Reply:
Done